### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by AnthonyBuisset**
Unit tests in CI are broken.
AMQP infrastructure tests are the problematic ones.
Some fail randomly with the following error message: 
```
ERROR infrastructure::amqp::bus: Lost connection to RabbitMQ: InvalidChannel(0)
```

Several modifications were made:
* upgrade lapin 
* upgrade RabbitMQ -> Rolled back as management UI seems broken
* create only one single connection to RabbitMQ -> Actual fix

### CODE REVIEW

**Review by ofux — APPROVED** [review]
After checking RabbitMQ best practices, it seems indeed that one connection per app is the recommended way. Well done

File: backend/event-listeners/src/models/github_repos_contributors/repository.rs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	-> Result<Vec<GithubUserId>>;
+}
+
+impl Repository for database::Client {
+	fn find_contributors_of_repo(
+		&self,
+		github_repo_id: &GithubRepoId,
+	) -> Result<Vec<GithubUserId>> {
+		let mut connection = self.connection()?;
+		let state = dsl::github_repos_contributors
```
Comment (AnthonyBuisset) [inline backend/event-listeners/src/models/github_repos_contributors/repository.rs:21]: not sure `state` is the right name, maybe `contributors` ?

File: backend/event-listeners/src/models/project_github_repos/repository.rs [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+use super::ProjectGithubRepo;
+
+pub trait Repository: database::ImmutableRepository<ProjectGithubRepo> {
+	fn find_projects_of_repo(&self, github_repo_id: &GithubRepoId) -> Result<Vec<ProjectId>>;
+}
+
+impl Repository for database::Client {
+	fn find_projects_of_repo(&self, github_repo_id: &GithubRepoId) -> Result<Vec<ProjectId>> {
+		let mut connection = self.connection()?;
+		let state = dsl::project_github_repos
```
Comment (AnthonyBuisset) [inline backend/event-listeners/src/models/project_github_repos/repository.rs:17]: same comment here

File: backend/event-listeners/src/models/projects_contributors/repository.rs [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+		Ok(())
+	}
+
+	fn unlink_project_with_contributor(
+		&self,
+		project_id: &ProjectId,
+		user_id: &GithubUserId,
+	) -> Result<()> {
+		let mut connection = self.connection()?;
+		connection.build_transaction().run::<_, diesel::result::Error, _>(|tx| {
```
Comment (AnthonyBuisset) [inline backend/event-listeners/src/models/projects_contributors/repository.rs:50]: what is the difference with `connection.transaction()` ?

**Pull request merged**